modules/hal_nordic/nrfx/nrfx_config_common.h

```diff
@@ -18,7 +18,7 @@
 
 /** @brief Symbol specifying minor version of the nrfx API to be used. */
 #ifndef NRFX_CONFIG_API_VER_MINOR
-#define NRFX_CONFIG_API_VER_MINOR 0
+#define NRFX_CONFIG_API_VER_MINOR 2
```

Comment:
That would make that one commit huge, combining multiple areas - is it really better?

Reply:
Yes, because otherwise the code does not build. Any commit that does not build is significantly slowing down bisect because you have to skip one, or more, commits which significantly increases the required number of checks. The required number of checks, when all commits in history do build correctly, is log2(N) where N is the number of commits in between good and bad revisions.

Reply:
The commits should really be:

- Update nrfx API version with all drivers adjusted for new API. No actual functional changes (if possible).
- Any subsequent commits are the functional changes related to things that were not possible with earlier nrfx version.